LoopKit/GlucoseUnitPublishing.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  GlucoseUnitPublishing.swift
+//  LoopKitUI
+//
+//  Created by Nathaniel Hamming on 2021-01-13.
+//  Copyright © 2021 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import HealthKit
+
+public protocol GlucoseUnitObserver {
```

Comment:
I'm wondering if we should be hand-rolling a special-case pub/sub system for this, as opposed to using [Combine](https://developer.apple.com/documentation/combine) now that it is part of Swift.  Thoughts, @ps2 ?